bin/templates/cordova/lib/build.js

```diff
@@ -148,10 +150,17 @@ module.exports.run = function (options, optResolvedTarget) {
             return;
         }
         return builder.build(opts).then(function () {
-            var apkPaths = builder.findOutputApks(opts.buildType, opts.arch);
-            events.emit('log', 'Built the following apk(s): \n\t' + apkPaths.join('\n\t'));
+            var paths;
+            if (opts.isBundle) {
+                paths = builder.findOutputBundles(opts.buildType);
+                events.emit('log', 'Built the following bundle(s): \n\t' + paths.join('\n\t'));
+            } else {
+                paths = builder.findOutputApks(opts.buildType, opts.arch);
+                events.emit('log', 'Built the following apk(s): \n\t' + paths.join('\n\t'));
+            }
+
             return {
-                apkPaths: apkPaths,
+                paths: paths,
```

Comment:
Technically speaking, changing the name of the key in the returned object is probably a breaking API change, and might require changes to stuff like the `run` command or cordova-lib and other tooling.

Reply:
I could revert `paths` back to `apkPaths` but the naming will be kind of misleading as it might not have APKs paths, but instead AAB paths now.

But I do agree it is a dangerous change. Perhaps I should just add a comment instead?

Reply:
How could this be used externally @dpogue? Isn't this just an internal API?

Reply:
Ah, you might be right. I see now that we wrap the result in `Api.prototype.build` before returning it to external callers, so hopefully it's fine.

We'll probably end up doing a major version bump to support Android Q later in the summer, so I don't necessarily think it's a blocker to make this change even it is potentially breaking.

Reply:
#729 milestone is for android 9, so I assumed this will be only included for android 9 as well.

Does this mean no changes is required for this at this time? Can I resolve conversation?

Reply:
No changes required :)